Tests/Command/ExplainAdminCommandTest.php

```diff
@@ -251,14 +251,16 @@ public function testExecute()
         $commandTester = new CommandTester($command);
         $commandTester->execute(array('command' => $command->getName(), 'admin' => 'acme.admin.foo'));
 
+        file_put_contents(getcwd().'/error.txt', $commandTester->getDisplay(true));
```

Comment:
What is this file? Will it always be generated? Have you considered using `tempnam` for that?